pkg/sfu/receiver.go

```diff
@@ -212,27 +202,13 @@ func NewWebRTCReceiver(
 	})
 
 	w.connectionStats = connectionquality.NewConnectionStats(connectionquality.ConnectionStatsParams{
-		CodecType:     w.kind,
-		GetDeltaStats: w.getDeltaStats,
-		GetLayerDimension: func(layer int32) (uint32, uint32) {
-			return w.GetLayerDimension(layer)
-		},
-		GetMaxExpectedLayer: func() *livekit.VideoLayer {
-			// find min of <expected, published> layer
-			maxExpectedSpatialLayer := w.streamTrackerManager.GetMaxExpectedSpatialLayer()
-			if maxExpectedSpatialLayer > w.maxPublishedLayer {
-				maxExpectedSpatialLayer = w.maxPublishedLayer
-			}
-			for _, layer := range w.trackInfo.Layers {
-				if maxExpectedSpatialLayer == utils.SpatialLayerForQuality(layer.Quality) {
-					return proto.Clone(layer).(*livekit.VideoLayer)
-				}
-			}
-
-			return nil
-		},
-		Logger:    w.logger,
-		CodecName: getCodecNameFromMime(w.codec.MimeType),
+		CodecType:           w.kind,
+		GetDeltaStats:       w.getDeltaStats,
+		IsDtxDisabled:       w.IsDtxDisabled,
+		GetLayerDimension:   w.GetLayerDimension,
+		GetMaxExpectedLayer: w.getMaxExpectedLayer,
```

Comment:
@shishirng I am moving these to `StreamTrackerManager` as this is needed by relay tracks as well. Sorry, I think I did not give you proper information before when you checked about relay tracks. I did not realise that down track will have to get the information from receiver.